vocab/template.html

```diff
@@ -0,0 +1,226 @@
+      superfluous.
+      See https://github.com/w3c/tr-design/pull/336
+      */
+      figure svg {
+          max-width: 100%;
+          margin: auto;
+          height: auto;
+      }
+    </style>
+  </head>
+  <body  typeof="owl:Ontology">
+    <section id="abstract">
+      <p>This document describes the <span property="dc:title" id="title"></span>, i.e.,
+         the <span property="dc:description" id="description"></span>.
+      </p>
+      <p>Alternate versions of the vocabulary definition exist in
+        <a rel="alternate" href="vocabulary.ttl">Turtle</a> and
+        <a rel="alternate" href="vocabulary.jsonld">JSON-LD</a>.
+      </p>
+      <dl>
+        <dt>Published:</dt><dd><time property="dc:date" id="time"></time></dd>
+        <dt>Version Info:</dt>
+        <dd>1.0</dd>
+        <dt id="see_also">See Also: </dt>
+       </dl>
+    </section>
+    <section id="sotd">
+      <p>
+This vocabulary has been defined by the
+<a href="https://www.w3.org/2017/vc/WG/">
+W3C Verifiable Credentials Working Group</a> and provides the official RDFS
+vocabulary for
+<a href="https://www.w3.org/TR/vc-bitstring-status-list/">
+Bitstring Status Lists</a>. Comments regarding this document are welcome.
+Please file issues directly on
+<a href="https://github.com/w3c/vc-bitstring-status-list/issues/">GitHub</a>, or
+send them to
+<a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a>
+(<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">
+subscribe</a>,
+<a href="https://lists.w3.org/Archives/Public/public-vc-comments/">
+archives</a>).
+      </p>
+    </section>
+    <section>
+      <h2>Specification of terms</h2>
+      <p>
+In general, the terms &mdash; i.e., the properties and classes &mdash; used in
+the VCDM are formally specified in Recommendation Track documents published by
+the <a href="https://www.w3.org/groups/wg/vc">W3C Verifiable Credentials Working
+Group</a> or, for some deprecated or reserved terms, in Reports published by the
+<a href="https://www.w3.org/groups/cg/credentials">W3C Credentials Community
+Group</a>. In each case of such external definition, the term's description in
+this document contains a link to the relevant specification. Additionally, the
+`rdfs:definedBy` property in the RDFS representation(s) refers to the formal
+specification.
+      </p>
+      <p>
+In some cases, a local explanation is necessary to complement, or to replace,
+the definition found in an external specification. For instance, this is so when
+the term is needed to provide a consistent structure to the RDFS vocabulary,
+such as when the term defines a common supertype for class instances that are
+used as objects of specific properties, or when
+<a href="https://www.w3.org/TR/rdf12-concepts/#section-rdf-graph">RDF Graphs</a>
+are involved. For such cases, the extra definition is included in the current
+document (and the `rdfs:comment` property is used to include them in the RDFS
+representations).
```

Comment:
Is this relevant? I.e., will this vocabulary ever deal with RDF Graphs? If not (which is probably the case) the whole section may be removed.

Reply:
Correct, this vocabulary will probably never deal w/ RDF Graphs. We can remove the statement about RDF Graphs.

Reply:
Mentions of RDF Graphs have been removed in the PR.